src/test/java/com/brew/oauth20/server/fixture/TokenRequestFixture.java

```diff
@@ -0,0 +1,42 @@
+package com.brew.oauth20.server.fixture;
+
+import com.brew.oauth20.server.data.enums.GrantType;
+import com.brew.oauth20.server.fixture.abstracts.Fixture;
+import com.brew.oauth20.server.model.TokenRequestModel;
+import com.brew.oauth20.server.testUtils.FakerUtils;
+import com.brew.oauth20.server.utils.StringUtils;
+import org.instancio.Instancio;
+import org.instancio.Model;
+
+import static org.instancio.Select.field;
+
+public class TokenRequestFixture extends Fixture<TokenRequestModel> {
+
+    private final GrantType[] defaultGrantTypeOptions = new GrantType[]{GrantType.refresh_token,
+            GrantType.authorization_code, GrantType.client_credentials};
+
+    public TokenRequestFixture() {
+        super();
+    }
+
+    public TokenRequestModel createRandomOne() {
+        return createRandomOne(this.defaultGrantTypeOptions);
+    }
+
+    public TokenRequestModel createRandomOne(GrantType[] grantTypeOptions) {
+        return Instancio.of(tokenRequest(grantTypeOptions))
+                .create();
+    }
+
+    private Model<TokenRequestModel> tokenRequest(GrantType[] grantTypeOptions) {
+        return Instancio.of(TokenRequestModel.class)
+                .supply(field(TokenRequestModel::getRefresh_token), () -> StringUtils.generateSecureRandomString())
```

Comment:
Buralarda StringUtils'e bağımlılık yaratmamalıyız bence test yazarken. fake stringler yeterli olur zaten case için refreshToken, clientId ve clientSecret yaratırken.